src/Components/pages/Live.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * ningjs - Live.js
+ */
+
+'use strict'
+
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+
+import 'styles/pages/article.scss'
+
+export default class Live extends Component {
+
+  componentDidMount() {
+    const options = {
+
+    }
+    videojs('ningjs_live', {flash: options, html5: options})
+  }
+
+  render() {
+    const { __, query } = this.props
+    return (
+      <div className='article'>
+        {/* video */}
+        <section id='hero'>
+          <div className='bg-container live'>
+            <div className='box'>
+              {/* masthead */}
+              <video
```

Comment:
Maybe we should add a count down like "Live stream start in 10 days" so it's less confusing?